src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,21 @@
-export const TodoInfo = () => {};
+import { UserInfo } from '../UserInfo';
+import { TodoWithUser } from '../../services/TodoWithUser';
+import cn from 'classnames';
+
+type Props = {
+  todo: TodoWithUser;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={cn('TodoInfo', {
+        'TodoInfo--completed': todo.completed === true,
+      })}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+      {todo.user !== undefined && <UserInfo user={todo.user} />}
```

Comment:
```suggestion
      {todo.user && <UserInfo user={todo.user} />}
```
BTW, you can use restructuring assignment to get all properties from `todo`. Same in other components